doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -22,15 +22,63 @@ as shown in :numref:`hybrid_scenario_on_nuc`.
    :local:
    :depth: 1
 
-Prerequisites
-*************
+.. rst-class:: numbered-step
+
+Set-up base installation
+************************
+
 - Use the `Intel NUC Kit NUC7i7DNHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i7dnhe.html>`_.
 - Connect to the serial port as described in :ref:`Connecting to the serial port <connect_serial_port>`.
 - Install Ubuntu 18.04 on your SATA device or on the NVME disk of your
   Intel NUC.
 
 .. rst-class:: numbered-step
 
+Prepare the Zephyr image
+************************
+
+Prepare the Zephyr kernel that you will run in VM0 later.
+
+- Follow step 1 from the :Ref:`using_zephyr_as_uos` instructions
+
+  .. note:: We only need the binary Zephyr kernel, not the entire ``zephyr.img``
```

Comment:
Step 1 in the referenced doc creates a ``zephyr.elf`` file.  How do we get the ``zephyr.bin`` file referenced in this next step?

Reply:
It's built at the same time and put in the same folder.